qemu_v8.mk

```diff
@@ -313,6 +379,17 @@ run: all
 
 QEMU_SMP ?= 2
 
+QEMU_MEM ?= 1057
+
+ifeq ($(XEN_BOOT),y)
+QEMU_MEM 	= 2048
+QEMU_SMP	= 4
+QEMU_XEN	?= -machine virtualization=true \
+		   -machine virt,gic-version=$(QEMU_GIC_VERSION) \
+		   -drive if=none,file=$(XEN_EXT4),format=raw,id=hd1 \
+		   -device virtio-blk-device,drive=hd1
+endif
```

Comment:
else
QEMU_SMP ?= 2
QEMU_MEM ?= 1057

Reply:
ok

Reply:
It would be nice to get rid of this `1057`, but perhaps in a separate PR.